app/inputs/active_admin/inputs/select_input.rb

```diff
@@ -2,4 +2,10 @@ class ActiveAdmin::Inputs::SelectInput < Formtastic::Inputs::SelectInput
   def input_html_options
     super.merge(data: { tags: !!@options[:tags] })
   end
+
+  def raw_collection
+    field_value = object.send(method)
+
+    !!@options[:tags] && field_value.present? ? (super.to_a << field_value).uniq : super
```

Comment:
Could there be a problem with overriding the `raw_collection` to `false` when the tags option is false? Shouldn't it fallback to whatever the default behaviour is?

Reply:
If I understood correctly, the second part of the ternary defaults to `super` so that should be the behaviour

Reply:
You're right, I mentally parsed the expression (incorrectly) as `A && (B ? C : D)` instead